src/hydration/user.ts

```diff
@@ -38,6 +40,7 @@ export const userHydration: Hydrate<ApiUser, HydratedUser> = {
     discriminator: (user) => user.discriminator,
     displayName: (user) => user.display_name!,
     relationship: (user) => user.relationship!,
+    relations: (user) => user.relations!,
```

Comment:
Relationship information is already given to the client by the server on individual user objects, this should instead just suppress the `skipping key` warning and not process the data at all.